app/javascript/packs/app/routes.js

```diff
@@ -1,6 +1,16 @@
+export const ROOT_PATH = "/";
+
 // activities
 export const ACTIVITIES_PATH = "/client_api/activities";
 
+// settings
+export const SETTINGS_PATH = "/settings";
+export const SETTINGS_ACCOUNT_PATH = "/settings/account";
+export const SETTINGS_TEAMS_PATH = "/settings/teams";
+export const SETTINGS_ACCOUNT_PROFILE_PATH = "/settings/account/profile";
+export const SETTINGS_ACCOUNT_PREFERENCES_PATH =
+  "/settings/account/preferences";
```

Comment:
move this: export const SETTINGS_ACCOUNT_PATH = "/settings/account";
+export const SETTINGS_TEAMS_PATH = "/settings/teams";
+export const SETTINGS_ACCOUNT_PROFILE_PATH = "/settings/account/profile";
+export const SETTINGS_ACCOUNT_PREFERENCES_PATH =
+  "/settings/account/preferences";

to a separate file called dom_routes.js

Reply:
We have routes.rb that contains all routes in one file. Should I separate those files as well?

Reply:
No

Reply:
Can you please don't mix the API endpoints and the dom-router paths!